### PR TITLE
Add note regarding identical release notes

### PR DESCRIPTION
**Pull request by TzviLipschitz**

### CODE REVIEW

**Review by bbyreddy — APPROVED** [review]

File: content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+change_type:
+  - value: change-inv-3bw8e
+    label: Announcement
+product_area: Edge
+component:
+  - value: component-
+    label: Edge Appliance VM
+build_artifact:
+  - value: tc-
+    label: edge
```
Comment (bbyreddy) [inline content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md]:
```suggestion
    label: Edge Appliance VM
```

File: content/change-logs/edge/edge-on-kubernetes-10-18-0-Release-identical-to-10-18.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+change_type:
+  - value: change-inv-3bw8e
+    label: Announcement
+product_area: Edge
+component:
+  - value: component-
+    label: Edge on Kubernetes
+build_artifact:
+  - value: tc-
+    label: edge
```
Comment (bbyreddy) [inline content/change-logs/edge/edge-on-kubernetes-10-18-0-Release-identical-to-10-18.md]:
```suggestion
    label: Edge on Kubernetes
```

File: content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md [hunk 3]
```diff
@@ -10,4 +10,4 @@ component:
     label: Edge Appliance VM
 version: 10.18.0
 ---
-The Cumulocity IoT Edge Appliance VM version of the 2024 release is identical to the version of the 10.18 release. For details on the release see the [10.18 release notes](https://cumulocity.com/releasenotes/release-10-18-0/edge-10-18-0/). You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/.
+The Cumulocity IoT Edge Appliance VM version of the 2024 release is identical to the version of the 10.18 release. For details on the release see the [10.18 release notes](https://cumulocity.com/releasenotes/release-10-18-0/edge-10-18-0/). You can find the documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/.
```
Comment (BeateRixen) [inline content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md:13]: @TzviLipschitz For the documentation, we would no longer link to the old guides but to the documentation in the 2024 documentation, wouldn´t we?

File: content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+date: 2024-05-08
```
Comment (BeateRixen) [inline content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md:2]: Do not add a date here but only in the 2024 release as this should not show up in the change logs for CD.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md (hunk 4 above)
Comment (TzviLipschitz) [reply content/change-logs/edge/edge-appliance-vm-10-18-0-Release-identical-to-10-18.md:2]: Leaving date in as discussed

**Comment by TzviLipschitz** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|release/y2024|[](https://github.com/SoftwareAG/c8y-docs/pull/2027)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**